.agents/skills/consolidation/nemoclaw-pr-review/SKILL.md

```
@@ -0,0 +1,497 @@
+  --json number,title,labels,updatedAt \
+  --search 'label:arch-improve OR label:refactor OR onboard OR shell OR type'
+```
+
+Only cite PRs/issues/modules you just verified are still relevant.
+
+#### How to phrase architectural feedback
+
+Use constructive suggestions, not demands. Examples:
+
+> 🟡 **Architectural note:** This adds a new function to `<large-file>` (currently `<N>` lines on this branch vs `<M>` on `origin/main`). Consider placing it in a focused module if a suitable current module exists, or extracting a new one. Not a blocker, but it helps the decomposition effort.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)